exporter/awsemfexporter/config.go

```diff
@@ -102,6 +108,8 @@ type MetricDescriptor struct {
 	Overwrite bool `mapstructure:"overwrite"`
 }
 
+var _ component.Config = (*Config)(nil)
```

Comment:
This will validate if the `Config` declared by `awsemfexporter Config` has all the declared function by the [interface](https://stackoverflow.com/a/27804417)